commands/mod.js

```diff
@@ -1,31 +1,39 @@
 //Make an announcement
 exports.announcement = function(message, args){
   var channelIdentifier = args.shift();
+  var channel;
   //Check if we are given a channelName or a channelID
+  logger.info(`Attempting to locate channel ${channelIdentifier}`);
   if(channelIdentifier.match(/^[0-9]+$/) != null){
     channel = global.util.getChannel({id: channelIdentifier, type: 'text'});
   }
+  else if(channelIdentifier.match(/^<#[0-9]+>$/) != null){

```

Comment:
Can you explain why we are using this weird regex everywhere? Seems overly complicated